packages/icon/test/icon-font.test.js

```diff
@@ -158,4 +158,65 @@ describe('vaadin-icon - icon fonts', () => {
       expect(icon.icon).to.equal('foo:bar');
     });
   });
+
+  describe('fontFamily', () => {
+    let icon;
+
+    it('should set font-family for the icon element', () => {
+      icon = fixtureSync('<vaadin-icon char="\\e900"></vaadin-icon>');
+      icon.fontFamily = 'My icons';
+      const fontIconStyle = getComputedStyle(icon, ':before');
+      expect(['"My icons"', 'My icons']).to.include(fontIconStyle.fontFamily);
+    });
+  });
+
+  // These tests make sure that the heavy container query fallback is only used
+  // when font icons are used.
+  (usesFontIconSizingFallback ? describe : describe.skip)('container query fallback', () => {
```

Comment:
Maybe I got it wrong, but to me, it seems that the comment is actually not true since the test suite is only running when the component is using the fallback, so it doesn't test that the fallback doesn't run when the browser supports the `container-type` property.

Maybe there should be a group of tests to check that the `--_vaadin-font-icon-size` variable is not being set when CSS Container is supported by the browser.

Reply:
This fix and the tests only concern browsers that need to use the fallback. But previously, the fallback was also active even if _font icons_ were not used. So when you use `<vaadin-icon icon="foo:bar">`, even on browsers that don't support container queries, there's no need to measure the offset size of the component nor set the `--_vaadin-font-icon-size`.